.devcontainer/devcontainer.json

```diff
@@ -19,7 +21,7 @@
     },
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "cd client && npm install"
+	"postCreateCommand": "docker image pull lostpixel/lost-pixel:v3.18.2 && cd client && npm install"
```

Comment:
Is there a better way of pulling the image preemptively?

Reply:
Why does it need to be pulled at this stage?

Reply:
If the image isn't available when running `npm run test:story` from docker in docker it fails because it doesn't have network access. So I could either grant network access or have the image already available, and because [the version of lost-pixel has telemetry we can't opt out of](https://github.com/lost-pixel/lost-pixel/issues/408) I thought it better to keep the network block. It's not a strong reason though.